chibuezeayogu/File-Transfer-App
# #164195450 Add and configure Rubocop gem

**What does the PR do?**
- This PR adds `RuboCop` rules to ensure that `RoR` style guide is adhered to.

**Description of Task to be completed?**
- Ensures that `RuboCop` is properly configured

**How should this be manually tested?**
- Clone the repository [File-Transfer-App](https://github.com/chibuezeayogu/File-Transfer-App.git) and
- Checkout to  this branch `chore-integrate-rubocop-164195450`
- run `rubocop` on the terminal
- ensure all `files are inspected and no offences detected`

**What are the relevant pivotal tracker stories?**
- [#164195450](https://www.pivotaltracker.com/story/show/164195450)

**Screenshots**
- None
